src/main/java/org/assertj/vavr/internal/Maps.java

```diff
@@ -161,14 +162,14 @@ public static Maps instance() {
      *
      * @param <K>     key type
      * @param <V>     value type
-     * @param info    contains information about the assertion
-     * @param actual  the given {@code Map}
-     * @param entries the given entries
+     * @param info    contains information about the assertion.
+     * @param actual  the given {@code Map}.
+     * @param entries the given entries.
      * @throws NullPointerException     if the given entries array is {@code null}.
      * @throws AssertionError           if the actual map is {@code null}.
      * @throws IllegalArgumentException if the given entries array is empty.
-     * @throws AssertionError           if the actual map does not contain the given entries with same order, i.e. the actual map
-     *                                  contains some or none of the given entries, or the actual map contains more entries than the given ones
+     * @throws AssertionError           if the actual map does not contain the given entries with same order, i.e. the actual map.
+     *                                  contains some or none of the given entries, or the actual map contains more entries than the given ones.
```

Comment:
To greedy ;] This is a single sentence! 😂

Reply:
Aaaaagrrh